src/lib/zod/index.ts

```diff
@@ -62,6 +62,13 @@ export const uploadCircleImageInput = z
   })
   .strict();
 
+export const vouchInput = z
+  .object({
+    nominee_id: z.number(),
+    circle_id: z.number(),
```

Comment:
Don't think `circle_id` is necessary, you can get the `circle_id` from the `nominee` object. 
theres no validation whether the nominee belongs to the `circle_id` currently